docs/source/intervention_models/crm_mgmt/tx.rst

```diff
@@ -0,0 +1,136 @@
+Blood pressure and lipid-lowering medications
+*********************************************
+
+blood pressure llt 
+
+
+Medication initiation:
+All simulants enrolled in the intervention initiate treatment (defined as initial fill of prescription(s))
+
+Adherence:
+All simulants get number from 0 to 1 drawn from non-uniform distribution of adherence in the general population [need to find]. Simulants with values >=0.8 are considered adherent and receive the full benefit of their medication.
```

Comment:
Do you need the full distribution of PDC?  I think just knowing who is >= 80% will be enough for your sim.